lib/router.js

```diff
@@ -0,0 +1,106 @@
+var fs = require("fs");
+
+module.exports = function(req, res) {
+
+  var path = req.url.split("/");
+  var fileName = path[path.length - 1];
+  var pathToFile = "./files/" + fileName + ".json";
+
+if (req.method === "GET" ) { // Read the file requested
+
+  fs.readFile(pathToFile, function(err, data) {
+  if (err) {
```

Comment:
Some of the indenting is off for this assignment. No need to change it for this assignment but I recommend keeping your indenting consistent for future assignments. It makes your code easier to maintain/read.